.builders/scripts/repair_wheels.py

```diff
@@ -279,7 +279,8 @@ def copy_filt_func(libname):
 
         copied_libs = delocate_wheel(
             str(wheel),
-            os.path.join(built_dir, wheel.name),
+            # Platform dependent wheels (e.g. confluence_kafka) must be published accordingly
+            os.path.join(built_dir, wheel.name.replace('universal2', os.uname().machine)),
```

Comment:
I know it's more verbose and that the risk of finding `universal2` anywhere but on the platform tag is minimal here, but maybe we can consider something like

```suggestion
            os.path.join(built_dir, str(wheel_name._replace(platform_tag=f'macosx_10_12_{os.uname().machine()}'))),
```

If we combine that with these two arguments [here](https://github.com/matthew-brett/delocate/blob/6a0b30ee392a610ef8f994047bdecb4a57c2c7e0/delocate/cmd/delocate_wheel.py#L119-L120), maybe we can get more guarantees that our platform tag is not a lie.

Reply:
Back to draft mode: investigating why `require_archs` doesn't work as expected (i.e. complains about the other arch missing)